### DISCUSSION TITLE
Management of purchase multiples

### DISCUSSION THREAD

**Discussion by abramofranchetti** [PrestaShop/PrestaShop#40167] [Ideas]
#### Problem:

Often it can happen that a product, in addition to having a minimum sales quantity, can only be sold in multiples of a certain quantity. ex a product may only be sold in multiples of 5, but with a minimum quantity of 10.
So at least 10 pieces, then 15, 20, 25 and so on.
At the moment Prestashop cannot handle this thing in Standard.
#### Describe the solution you'd like

It would be useful if a "sales multiple" field in the product could be managed in which the purchase multiples for a product can be defined.
This property should then set the "step" property of the jquery touchspin inputs in the frontoffice.
it would be nice if this feature was a standard prestashop feature
#### Additional context

At the moment, in the frontoffice, the purchase quantity is managed with the "jquery touchspin" component. The component would manage the "steps" as standard, but there is no possibility without a manual modification of the prestashop core, or subsequent implementations, to set the "step" property in the jquery touchspin inputs.

There are some modules that attempt to handle this, but they often use the "minimal_quantity" field, thus preventing you from differentiating the concepts of minimum quantity and multiple of sale. They also often don't work properly on all prestashop themes / sections.

**Comment by florine2623** [comment]
Hello @abramofranchetti ,

Modules that do this feature exist on the addons marketplace. 
For example, this one : https://addons.prestashop.com/en/cross-selling-product-bundles/25969-sell-by-x-units.html#specifications

I'll ping PS Product team to have their point of view on that.
Thanks!

**Comment by florine2623** [comment]
ping @PrestaShop/product-team 

What do you think about this feature ?

Thanks!

**Comment by Hlavtox** [comment]
I had a client with this need. Implemented this via a custom message on the product page and verifications on cart + order page. Works good. It does not interfere with the quantity process at all. Works by blocking the checkout and saying something like 

```
Please modify the quantity of following products, so they fit the package quantity. Thank you!
XXX either 12 or 24 pieces
```
